### ISSUE TITLE
KeyError: 'DELIVERED_READ'

### ISSUE THREAD

**Issue by asn-d6** [mikeperry-tor/vanguards#2]
Got this today when visiting an HS with a client:
```
NOTICE[Mon May 28 13:31:54 2018]: Connected to Tor version 0.3.4.0-alpha-dev (git-c27c86d113108107)
...
DEBUG[Mon May 28 13:32:16 2018]: 650 CIRC_BW ID=27 READ=389 WRITTEN=10 TIME=2018-05-28T10:32:16.010609
Exception in thread Event Notifier:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/usr/lib/python2.7/threading.py", line 754, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/f/.local/lib/python2.7/site-packages/stem/control.py", line 953, in _event_loop
    self._handle_event(event_message)
  File "/home/f/.local/lib/python2.7/site-packages/stem/control.py", line 3727, in _handle_event
    listener(event_message)
  File "/home/f/Computers/tor/prop247/vanguards/src/vanguards/bandguards.py", line 122, in circbw_event
    delivered_read = int(event.keyword_args["DELIVERED_READ"])
KeyError: 'DELIVERED_READ'
```

**Comment by mikeperry-tor** [comment]
Looks like you're using a tor master built before #25903 was merged. This used to work for you because the vanguards script used to explicitly check for those fields on every CIRC_BW event. In vanguards commit ec8694ba0f0517a5e3bfd41f89b0d961eac5274e I removed the check on every CIRC_BW event and switched to a version comparison against 0.3.4.0-alpha for efficiency, since #25903 was merged prior to the first alpha release.

If you update your Tor the issue should go away. I don't think this is a bug.
